rules.json

```
@@ -2,16 +2,167 @@
+    "• For the sake of everyone, it is only fair and respectful that TXDPSRP is an ENGLISH only network. Please refrain from speaking any other languages than English in any text/voice channels.",
+    "Advertising",
+    "• Do NOT have a social media link, IP, or another FiveM server in your discord status message.",
+    "• Do NOT advertise another server in our text/voice channels. You may advertise another server in Private Direct Messages, but ONLY IF THEY ASK FOR IT. Failure to follow this rule will result in a permanent ban.",
+    "• Similarly, do not advertise your YouTube channel or Social Media to any users unless they ASK FOR IT.",
+    "Notes",
+    "Anything you say to users through private messages on any application can be punishable on the Discord Server as well. That includes harassment, unnecessary swearing, sending random links, etc. Message a staff member with solid evidence, such as screenshots, videos, etc, if you feel someone should be punished for something they said through Private Messages.",
+
+    "We, The TXDPSRP Network also strictly follow the Discord Terms of Service (TOS) and their Guidelines! If you are found breaking either of these you will be BANNED without notice and/or explanation",
+            
+
+
+
+
+
+
+            
+
     },
     {
-        "title": "Roleplay Standards",
-        "isChild": true,
+        "title": "Miscellaneous Rules",
+        "isChild": false,
         "rules": [
-            "You must realistically roleplay at all times. Fail-RolePlay (FailRP), Random Deathmatches (RDM), Vehicle Deathmatches (VDM), Meta-Gaming, Noclipping, Teleporting, Godmode, Invisibility and/or any type of so-called “Trolling” or “No Intent to RP (NITRP)” will not be tolerated.",
-            "Do NOT intentionally bait Law Enforcement Officers, also known as Copbaiting; for example, do NOT try to intentionally get a Law Enforcement Officer to initiate a traffic stop on you.",
-            "Do NOT interfere with any roleplay that you are not apart of.",
-            "Do NOT perform any Erotic RP. This includes roleplay containing sexism and nudity.",
-            "Whatever else would go here, this is just an example...."
+     "• Do NOT sell or purchase anything of some sort that displays the TXDPSRP Logo, unless approved, distributed, and/or promoted by Head Administration.",
+     "• Do NOT create any discord/teamspeak servers in the name of TXDPSRP, unless approved by the founder.",
+     "•  Do NOT distribute any files or documents which may be harmful to a user.",
+     " • Do NOT distribute or plagiarize any documents created within The TXDPSRP Network.",
+     "• Do NOT create any official TXDPSRP social media accounts. This is handled by our Social Media Team.",
+     "• Do NOT attempt to sell any in-game items for profit in real-life. ",
+     "• Maturity must be maintained at ALL TIMES, even when a Staff Member is not in the channel.",
```

Just remove the comma here, the ``]`` is present here.
```suggestion
     "• Maturity must be maintained at ALL TIMES, even when a Staff Member is not in the channel."
```